lib/db_connection.ex

```diff
@@ -410,6 +414,20 @@ defmodule DBConnection do
   requests before they are sent to the database, which would
   otherwise increase the burden on the database, making the
   overload worse.
+
+  ## Connection listeners
+
+  The `:connection_listeners` option allows one or more processes to be notified
+  whenever a connection is connected or disconnected. A listener may be a remote
+  or local PID, a local port, a locally registered name, or a tuple in the form
```

Comment:
It can't really be a port as we don't send messages required by the port format. :)

Reply:
I've removed port from the list in 0eae12ea98a49120a3634a3a9afc1e9a186701e9.